release-1.8/release_notes_draft.md

```diff
@@ -59,7 +59,15 @@ on supporting the broadest set of workload types, including hardware and perform
 sensitive workloads such as data analytics and deep learning, while delivering
 incremental improvements to node reliability.
 
+### SIG Storage
 
+[SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage) is
```

Comment:
For the 1.8 release, SIG Storage extends the Kubernetes storage API beyond making volumes available to volume resizing and snapshotting.